contracts/Ethereum_SpokePool.sol

```diff
@@ -25,7 +29,7 @@ contract Ethereum_SpokePool is SpokePool, Ownable {
      **************************************/
 
     function _bridgeTokensToHubPool(RelayerRefundLeaf memory relayerRefundLeaf) internal override {
-        IERC20(relayerRefundLeaf.l2TokenAddress).transfer(hubPool, relayerRefundLeaf.amountToReturn);
+        IERC20(relayerRefundLeaf.l2TokenAddress).safeTransfer(hubPool, relayerRefundLeaf.amountToReturn);
```

Comment:
Can we put this in a separate PR since it's pretty distinct from the pause functionality?